WordPress/WordPressDraftActionExtension/WordPressDraft-Lumberjack.m

```diff
@@ -1,6 +0,0 @@
-@import CocoaLumberjack;
-
-
-/// CocoaLumberjack Loglevel: Let's nuke this, whenever possible.
-///
-DDLogLevel ddLogLevel = DDLogLevelInfo;
```

Comment:
I nuked these 😄 
Couldn't find where they were used if anywhere. I assumed it wasn't needed in app extensions anymore.